cmd/core/app/options/options.go

```diff
@@ -122,7 +120,6 @@ func (s *CoreOptions) Flags() cliflag.NamedFlagSets {
 		"The re-sync period for informer in controller-runtime. This is a system-level configuration.")
 	gfs.Float64Var(&s.QPS, "kube-api-qps", s.QPS, "the qps for reconcile clients. Low qps may lead to low throughput. High qps may give stress to api-server. Raise this value if concurrent-reconciles is set to be high.")
 	gfs.IntVar(&s.Burst, "kube-api-burst", s.Burst, "the burst for reconcile clients. Recommend setting it qps*2.")
-	gfs.StringVar(&s.LeaderElectionResourceLock, "leader-election-resource-lock", s.LeaderElectionResourceLock, "The resource lock to use for leader election")
```

Comment:
Why is the flag removed？

Reply:
@phantomnat can you answer this?

Reply:
From the [doc](https://github.com/kubernetes-sigs/controller-runtime/blob/3e6681045387530b740e4b35a96a6b7fe590718a/pkg/manager/manager.go#L183), I thought that we might not need to custom this. 
But feel free to bring it back.

Reply:
@StevenLeiZhang Is it necessary to bring it back?